Sources/Bow/Typeclasses/Traverse+Scan.swift

```diff
@@ -0,0 +1,131 @@
+import Foundation
+
+public extension Traverse {
```

Comment:
Move these extensions to the `Traverse.swift` file to keep them together and avoid too much segregation.